typespec-tests/src/main/java/com/cadl/longrunning/LongRunningClient.java

```diff
@@ -205,11 +210,11 @@ public JobResult getJob(String id) {
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return the {@link SyncPoller} for polling of status details for long running operations.
+     * @return the {@link SyncPoller} for polling of long-running operation.
      */
     @Generated
     @ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)
-    public SyncPoller<PollOperationDetails, JobResultResult> beginCreateJob(JobData jobData) {
+    public SyncPoller<JobResult, JobResultResult> beginCreateJob(JobData jobData) {
```

Comment:
Diff here is because a slight change in logic. `JobResult` does not refer `Azure.Core.Foundations.OperationStatus`. So, here it does not use Java core class.